akirachix/Upile-Frontend
# adds sidebar

**What this PR does?**

    **What is this?**
    This PR introduces a new SidebarNav component for the LostLocate dashboard, providing easy navigation across different dashboard sections.

    **Why this?**
    The sidebar enhances the user experience by providing a clear, accessible way to move between pages, particularly for users accessing the application on a wide range of devices, including desktops and tablets.

    **How are you doing this?**

-         Created a new React component SidebarNav that includes navigation links to dashboard features such as Home, Missing Persons, Reports, and more.
-         Utilized react icons to ensure the sidebar is visually engaging and user-friendly.
-         Styled the sidebar with custom fonts and CSS to match the project’s overall theme.

**Updates:**

-     Added the SidebarNav component.
-     Integrated lucide-react for icons.
-     Enhanced responsiveness for multiple screen sizes (desktop, tablets).
-     Applied styles using Poppins font for consistency.

**Type of change**


**How can it be tested?**

-     Pull the branch and run the project locally.
-     Navigate through the dashboard using the new sidebar links.
-     Resize the screen or use browser dev tools to test responsiveness for various devices such as Nest Hub, iPad Pro, and desktop.
-     Check if all icons and links function as expected.

**Screenshots**
![Screenshot from 2024-09-18 14-41-47](https://github.com/user-attachments/assets/b1ba7acc-2dc6-4755-9201-d2e4cff8869f)

**Where is it documented?**
https://wambuinatalygakenye.atlassian.net/browse/UP-76?atlOrigin=eyJpIjoiOTU0ZDA5MjVhMmUxNDNlMGEwYzY0NmZhYWE2NzU5YTEiLCJwIjoiaiJ9
